util/reflect.go

```diff
@@ -444,6 +444,12 @@ func forEachFieldInternal(ni *NodeInfo, in, out interface{}, iterFunction FieldI
 		return nil
 	}
 
+	// If the field is an annotation, then we do not process it any further, including
+	// skipping running the iterFunction.
+	if IsYgotAnnotation(ni.StructField) {
```

Comment:
Should we say this in the doc comment on this function? Since it no longer does its work "ForEachField".

Reply:
Added a comment here and a shorter one in `forEachInternal` to cover this. Thanks.